### DISCUSSION TITLE
editor scroll speed

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#22424] [Ideas]
on some computer setups the editor scrolls way too fast or too slow, as sometimes the ratio of scrolling normally to scrolling in editor isn't comfortable. an option to change how fast the editor scrolls would be so useful.

**Comment by peppy** [comment]
Can you explain where you are attempting to scroll? There's multiple areas of the editor which can be scrolled. Also, if it's too fast or slow, then it should also be the same game-wide (ie. scrolling in settings). Can you confirm this?

**Reply by Axilotl17** [reply]
specifically in the **timeline**; It does scroll a bit fast game-wide, however it is only an issue in the editor in the timeline. Everywhere else it isn't a big problem.

**Reply by peppy** [reply]
Can you please let us know what hardware you're using? Potentially a high precision trackpad (aka microsoft surface or apple laptop)?

**Reply by Axilotl17** [reply]
I'm using arch linux w/ framework laptop; however, I think its not that the scroll speed should be slower, just it should be configurable.

**Reply by bdach** [reply]
Having it be configurable is the absolute last resort. It should be a sane default for all platforms first and foremost.
